examples/user_guide/Interactive_Hover_for_Big_Data.ipynb

```diff
@@ -0,0 +1,184 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "basic_datashade",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "rasterized = rasterize(points)\n",
+    "shaded = datashade(points)\n",
+    "rasterized + shaded"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "aggregator_explanation",
+   "metadata": {},
+   "source": [
+    "## What is the difference between an Aggregator and a Selector?\n",
+    "\n",
+    "An aggregator in Datashader is a function that combines data points that fall into the same pixel.\n",
+    "\n",
+    "For example:\n",
+    "\n",
+    "- `ds.count()`: Calculate the number of points (does not depend on any specific column).\n",
+    "- `ds.mean(\"s\")`: Calculate the mean of column `s` for all points in the pixel.\n",
+    "- `ds.max(\"s\")`: Select the minimum value of column `s` for all points in the pixel.\n",
+    "\n",
+    "It is important to note the difference between the first two aggregators and the last one. The first two calculate a value, whereas the last one selects a row; this is how a `selector` is defined. \n",
+    "This means that a `selector` is a subset of an `aggregator`, which only selects a value and not doing any calculating.\n",
+    "Currently the following selectors are supported: `ds.min`, `ds.max`, `ds.first`, and `ds.last`.\n",
+    "\n",
+    "Because a `selector` is a subset, it can also be used as an aggregator in an operation."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "custom_aggregator",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "rasterized_mean = rasterize(points, aggregator=ds.mean(\"s\"))\n",
+    "rasterized_max = rasterize(points, aggregator=ds.max(\"s\"))\n",
+    "rasterized_mean + rasterized_max"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "server_side_hover_intro",
+   "metadata": {},
+   "source": [
+    "## Server-side HoverTool\n",
+    "\n",
+    "The key idea behind the server-side `HoverTool` is:\n",
+    "\n",
+    "1. **Hover event**: When a user hovers over a plot, the pixel coordinates are sent to the server.\n",
+    "2. **Data lookup**: The server uses these coordinates to look up the corresponding aggregated data from the pre-computed dataset.\n",
+    "3. **Update display**: The hover information is updated and sent back to the front end to display detailed data.\n",
+    "\n",
+    "This design avoids sending all the raw data to the client and only transmits the necessary information on-demand. You can enable this by adding a `selector` to a `rasterize` or `datashade` operation. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "server_hover_example",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "rasterized_with_selector = rasterize(points, aggregator=ds.mean(\"s\"), selector=ds.min(\"s\"))\n",
```

Comment:
Small feedback, the hover tool doesn't let the user know how the data was aggregated/selected, i.e. I don't know if I'm looking at the average/min/max/etc.

<img width="802" alt="image" src="https://github.com/user-attachments/assets/4c998708-d1bb-4bf4-a53c-f33311a908b9" />

Can users update the hover name to make this clearer? If so, I'd suggest showing this in the guide, I was pretty confused with `x_y s` and `s` for example.

Reply:
Maybe also precise somewhere than with `selector` and by default the data of all the vdims is displayed in the hover for the selected row.

Reply:
> Small feedback, the hover tool doesn't let the user know how the data was aggregated/selected, i.e. I don't know if I'm looking at the average/min/max/etc.

Have added some text to this. 

> Can users update the hover name to make this clearer? If so, I'd suggest showing this in the guide, I was pretty confused with x_y s and s for example. 

Not currently. They can hide columns with `hover_tooltips`; I have added an example of how to do this in the notebook. I have also added your request to rename columns to https://github.com/holoviz/holoviews/issues/6515 item 3. 

> Maybe also precise somewhere than with selector and by default the data of all the vdims is displayed in the hover for the selected row.

This should also now be clarified in the notebook.

Reply:
This discussion has moved to https://github.com/holoviz/holoviews/pull/6568